src/Controller/EventController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace App\Controller;
+
+use App\Repository\EventRepository;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use App\Entity\Event;
+use App\Entity\EventCategory;
+use App\Entity\EventType;
+
+
+class EventController extends BaseController
+{
+    const ITEMS_PER_PAGE = 5;
+
+    /**
+     * @Route("/event/list", name="list_events")
+     * @Template("event/event_list.html.twig")
+     */
+    public function listEvent(Request $request, EventRepository $eventRepository)
+    {
+
+        $page = $request->query->get('page', 1);
+        $limit = self::ITEMS_PER_PAGE;
```

Comment:
@daria-sieroshtan 
This var is only used once, when you calculate the number of pages
Hence you do not need var - put const directly there